sdg/Indicator.py

```diff
@@ -234,3 +234,65 @@ def translate_data(text):
 
         # Finally place the translation for later access.
         self.translations[language] = indicator
+
+    def is_complete(self):
+        """Decide whether this indicator can be considered "complete".
+
+        Returns
+        -------
+        boolean
+            True if the indicator can be considered "complete", False otherwise.
+        """
+        # First, check for an open-sdg-style "reporting_status" metadata field,
+        # for a value of "complete".
```

Comment:
Please focus on making the code readable, before considering writing comments. I think for this specific example, the code below is pretty well self-documented.

Reply:
@SavvasStephanides I wanted to make clear that we are looking for something specific to open-sdg. Remember that sdg-build (in theory at least) is not limited to open-sdg stuff. So, normally I would refrain from putting open-sdg-specific code in Indicator.py. But since it's somewhat harmless and time is short, I decided to go ahead and do that, along with a comment to make clear what's going on.